includes/definitions/discovery/supermicro-bmc.yaml

```diff
@@ -0,0 +1,102 @@
+mib: ATEN-IPMI-MIB
+modules:
+    os:
+        version:
+            - ATEN-IPMI-MIB::bmcMajorVesion.0
+            - ATEN-IPMI-MIB::bmcMinorVesion.0
+        version_template: "FW {{ ATEN-IPMI-MIB::bmcMajorVesion.0 }}.{{ ATEN-IPMI-MIB::bmcMinorVesion.0 }}"
+        features: ATEN-IPMI-MIB::biosVesion.0
+        features_template: "Bios {{ ATEN-IPMI-MIB::biosVesion.0 }}"
+    sensors:
+        pre-cache:
+            data:
+                -
+                    oid:
```

Comment:
Pretty sure you don't have to precache the sensortable

Reply:
hi @Jellyfrog 
thank you for review
removed pre-cache entry

i tested it with & without, but in this case could not see any difference

same time for both case
```
librenms@nms:~$ time ./discovery.php -d -h 100010 -m sensors > /tmp/out1

real    0m42.773s
user    0m1.497s
sys     0m0.882s
librenms@nms:~$ 

librenms@nms:~$ time ./discovery.php -d -h 100010 -m sensors > /tmp/out2

real    0m42.779s
user    0m1.455s
sys     0m0.951s
```

same output 
```
librenms@nms:/tmp$ ls -l out*
-rw-r--r-- 1 librenms librenms 478805 Jan 16 06:07 out1
-rw-r--r-- 1 librenms librenms 478805 Jan 16 06:09 out2
```